src/useCookieConsent.ts

```diff
@@ -16,19 +16,27 @@ export const useCookieConsent = (
 ): CookieConsentHookState => {
   const initialConsent: CookieConsent =
     Cookies.getJSON(COOKIE_CONSENT_KEY) ||
+    JSON.parse(localStorage.getItem(COOKIE_CONSENT_KEY)) ||
+    JSON.parse(sessionStorage.getItem(COOKIE_CONSENT_KEY)) ||
     options?.defaultConsent ||
     DEFAULT_CONSENT;
```

Comment:
There should be 2 different initial consents which are handled based on whether the `options.storage` is provided or not. Since you could have 2 hooks, but one storing the values in `localStorage` and other one storing it in the cookies